src/mapping/OurZora.ts

```diff
@@ -0,0 +1,36 @@
+import { Transfer } from "../../generated/templates/NftContract/ERC721";
+import { Nft, NftContract } from "../../generated/schema";
+import { fetchName, fetchSymbol, handleTransfer } from "./mapping";
+import { OurZoraNFT as OurZora } from "../../generated/OurZora/OurZoraNFT";
+import { OURZORA_NFT, ZERO_ADDRESS } from "../constants/index";
+import { Address, BigInt, log } from "@graphprotocol/graph-ts";
+
+export function handleTransferOurZora(event: Transfer): void {
+  let address = event.address.toHexString();
+  if (NftContract.load(address) == null) {
+    let nftContract = new NftContract(address);
+    nftContract.name = fetchName(event.address);
+    nftContract.symbol = fetchSymbol(event.address);
+    nftContract.platform = "OurZora";
+    nftContract.save();
+  }
+
+  handleTransfer(event);
```

Comment:
In Zora, there are two distinct URI fields: tokenURI and metadataURI:
https://zora.engineering/protocol/smart-contracts

> This protocol deviates from the ERC-721 in that the tokenURI does not point to a valid ERC721 Metadata JSON Schema as defined in the EIP. In order to support integrity checks when updating the tokenURIs, the content and metadata of a piece of media are split into tokenURI and metadataURI, respectively.

We can leave URI mapping as it is for now, but in the future we will need to rethink how we handle URI across different protocols